project/templates/register.php

```diff
@@ -6,16 +6,51 @@
 </head>
 <body>
 
-/*
-    Register form here, on submit POST on single, target singleRegistrationController.php
-*/
+<?php
+// ERROR SHOWING
+include_once "../constants/registerConstants.php";
+
+$errorMessage = getErrorMessage();
+if ($errorMessage != null) {
+    echo $errorMessage;
+}
+?>
+
+<form method="POST" action="../controller/singleRegistrationController.php">
+
+    <input type="email" id="email" name="email" placeholder="Email" required>
+    <input type="text" id="name" name="name" placeholder="Meno" required>
+    <input type="text" id="surname" name="surname" placeholder="Priezvisko" required>
+    <input type="password" id="password" name="password" placeholder="Heslo" required>
+    <input type="password" id="password-confirm" name="password-confirm" placeholder="Potvrdťe heslo" required>
+    <input type="submit" value="Registrovať účet">
 
-/*
-    Mass register will probably be in ADMIN home menu, so no need to have it here
-*/
+</form>
 
 <?php
 
+function getErrorMessage()
+{
+    if (isset($_GET["status"])) {
+
+        $status = $_GET["status"];
+
+        if ($status == ERROR_PASSWORD_MISMATCH) {
```

Comment:
This could be switch for better clarity maybe?

Reply:
the constants contain strings, if its possible to switch string then sure, otherwise i have to change their types (i considered numbers such as -2, -3, -4 ...) 
what should be my course of action?

Reply:
I think its much better from readability perspective to have switch. Also might be slightly faster, because multiple if's are not evaluated. But that would need some testing.

Reply:
fair enough, i'm going to refactor and request another pull

Reply:
If possible, just commit and push refactored code to this pull request. I think there is no need to create another pull request, just for small fix.

Reply:
commited and pushed, current pull request should be updated